docs/ui/components/Table/Table.tsx

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/react';
+import { theme, borderRadius } from '@expo/styleguide';
+import React, { PropsWithChildren } from 'react';
+
+import { TableHead } from './TableHead';
+import { TextAlign } from './types';
+
+type TableProps = PropsWithChildren<{
+  headers?: string[];
+  headersAlign?: TextAlign[];
+}>;
+
+export const Table = ({ children, headers = [], headersAlign = [] }: TableProps) => (
+  <div css={tableWrapperStyle}>
+    <table css={tableStyle}>
+      {headers.length ? (
+        <>
+          <TableHead headers={headers} headersAlign={headersAlign} />
+          <tbody>{children}</tbody>
+        </>
+      ) : (
+        children
+      )}
+    </table>
+  </div>
+);
+
+const tableWrapperStyle = css({
+  borderWidth: 1,
+  borderStyle: 'solid',
+  borderColor: theme.border.default,
+  borderRadius: borderRadius.large,
+  overflow: 'hidden',
+  marginBottom: '1rem',
```

Comment:
Maybe we should move this to the markdown components, instead of in the table?

Reply:
IIRC, when I write this code, I had thought that due to design (rounded corners) those component should never be used separately, otherwise, in both cases the design will be off. So I think that's the reason why I have left wrapper in there and do not extract it to the separate component.

Reply:
I think when using this inside another reusable component, we should be able to add them manually. And if you use this component inside markdown, markdown adds a paragraph around it by default (and thus margins). We can keep an eye on this and merge it as is, we can remove it later if we need to.